crates/story/src/scrollable_story.rs

```diff
@@ -285,14 +291,17 @@ impl Render for ScrollableStory {
                     .border_1()
                     .border_color(cx.theme().border)
                     .w_full()
-                    .max_h(px(400.))
+                    // .max_h(px(400.)) // move the max_h to the next level
                     .min_h(px(200.))
                     .child(
                         v_flex()
                             .p_3()
                             .w(self.test_width)
                             .id("test-1")
-                            .scrollable(Axis::Vertical)
+                            // .scrollable(Axis::Vertical)
```

Comment:
This is not needed to change, that container only have vertical scrollbar is by design, it not a bug.

Reply:
https://github.com/user-attachments/assets/645c962f-7958-4df7-8c23-60861f09c466

Got it, I changed it to follow the "self-axis" setting. When the setting is "Horizontal", you can scroll up/down only, and the vertical scroll bar is invisible, but the horizontal scroll bar is visible. I'm not sure if this will cause confusion or not.